coilsnake/assets/modulelist.txt

```diff
@@ -1,5 +1,6 @@
 common.UsedRangeModule
 common.PatchModule
+eb.AnimationModule
```

Comment:
the idea behind the ordering of this file is that all of the modules that compile quickly should go near the top, so that compilation errors can be caught more quickly. could you please move this somewhere below the TownMapIconModule?

Reply:
Done.